<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3091</width>
    <height>1862</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>514</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Complex Function Grapher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/resources/icons/Final.ico</normaloff>:/icons/resources/icons/Final.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ComplexView" name="complexView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>3071</width>
      <height>1791</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3091</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save Image</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionBMP">
   <property name="text">
    <string>BMP</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComplexView</class>
   <extends>QGraphicsView</extends>
   <header>complexview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>toolBar</sender>
   <signal>actionTriggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>toolbarTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>829</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>menuTriggered(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1052</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>1087</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>drawCanvas()</slot>
  <slot>toolbarTriggered(QAction*)</slot>
  <slot>menuTriggered(QAction*)</slot>
 </slots>
</ui>
